Source/MonoGame.Extended/Timers/ContinuousClock.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MonoGame.Extended.Timers
+{
+    public class ContinuousClock : IGameComponent
```

Comment:
>  I can just go ahead and remove the IGameComponent altogether.

I'd rather not use IGameComponent at all. At least until there's an obvious reason to do so. I'm a strong believer in keeping code as simple as possible. If we decide to support XNA game components we'll do it across the entire project as a separate issue.